### PR TITLE
Task/cooks 377 add community characteristics frontend

### PR DESCRIPTION
**Pull request by nathanfranklin**
## Overview: ##

* Click on the "Community Characteristics" link to open dialog
* Add new pie chart to backend
* Increase the width of plotly charts to 100%

## Related Jira tickets: ##

* [COOKS-377](https://jira.tacc.utexas.edu/browse/COOKS-377)

## Testing Steps: ##

1. View /protx/dash/analytics 
2. Click on a county
3. Click on "Community Characteristics"

-----

1. View /protx/dash/maltreament and demographics
2. Click on a county
3. ensure spacing of plotly charts as we have changed the width of them.

## UI Pho
Screenshot 2023-01-16 at 9 52 16 PM
Screenshot 2023-01-16 at 9 36 11 PM
Screenshot 2023-01-16 at 9 35 37 PM
tos:

## Notes: ##

TODO
- [x] add ethnicity piechart
- [ ] add frontend unit tests (or JIRA issue)
- [x] order and layout feedback from scrum meeting

### CODE REVIEW

File: protx-client/src/components/ProTx/components/charts/MainPlot.css [hunk 1]
```diff
@@ -15,4 +15,6 @@
 }
 .main-plot {
   margin: 0 1vw;
+  width: 100%;
```
Comment (taoteg) [inline protx-client/src/components/ProTx/components/charts/MainPlot.css:18]: This should be restricted to `width: 96%;`. 
- If it is above 96%, it generates the horizontal scrollbar on the iframe element.
-- We should be able to improve this when I get the new iframe template PR in.
- If it is above 98%, it cuts off the text labels on maltreatment plot on the right-side.

Given the minute difference between 96 and 98%, I recommend the 96% to eliminate the horizontal scrolling issue and clean up the entire page layout (again, pending a better template and unified way to handle iframe embedding).

Comment (taoteg) [reply protx-client/src/components/ProTx/components/charts/MainPlot.css:18]:
```suggestion
  width: 96%;
```

**Review by taoteg — COMMENTED** [review]
Other than the code suggestion I made, looks and behaves fine.

**Review by taoteg — APPROVED** [review]
LGTM!

**Comment by nathanfranklin** [comment]
@taoteg , merging and will bump plotly libs in a follow on pr.

**Pull request merged**